Require phpBB 3.1.2 minimum version

iMattPro opened the pull request.

iMattPro committed:
Require phpBB 3.1.2 minimum version

Wolfsblvt commented on ext.php:
```diff
@@ -20,4 +20,19 @@
+	public function is_enableable()
+	{
+		$config = $this->container->get('config');
+		return version_compare($config['version'], '3.1.2', '>=');
```
Seems good for me. This check will prevent many help questions, I would guess.
But do you want to use hardcoded version number here? Maybe it's easier, but I tend to forget that and just change composer.json :D We could read the composer metadata here, but We would have to split the phpbb version string. Hmh.
Maybe it's easier to just hardcode it, yeah.

iMattPro committed:
Add some interface for design guidance

iMattPro committed:
Update composer and travis files

iMattPro committed:
Disable ICC profile checking

This causes travis to fail because it is checking in vendor/phing and finding bad images. phpBB will need to be updated to not check extension vendor dir before this can be enabled again.
